recipe/meta.yaml

```diff
@@ -29,18 +29,15 @@ requirements:
     - python >=3.9
     - setuptools >=51.0.0
   run:
-    - __{{ noarch_platform }}
+    - __unix          # [unix]
```

Comment:
i guess at some point, the linter complained loudly about the presence of `# [platform]` selectors in `noarch: python` recipes.